src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.abstractandenum.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final int colorIndex = new Random().nextInt(Color.values().length);
+
+    public String getRandomColor() {
+        return Color.values()[colorIndex].name();
```

Comment:
This method should return different random color each time.
And this is what you have
![image](https://user-images.githubusercontent.com/20326858/214667117-ccdd24ba-188e-4142-b68d-c178b4cadfff.png)